buffalo/data/__init__.py

```diff
@@ -1,6 +1,9 @@
 # -*- coding: utf-8 -*-
 from buffalo.misc import aux
 
+from .mm import *
+from .stream import *
```

Comment:
아래처럼 전체 경로를 써주면 어떨까요?

```suggestion
from buffalo.data.mm import *
from buffalo.data.stream import *
```

Reply:
여기는 internal이라 relative import가 더 좋아 보입니당

Reply:
아 그러네요 다른 오픈소스 찾아보니 인터널은 relative로 가는군요? 그러면 지금 해주신대로 가시죠!